docs/vectorizer.md

```diff
@@ -69,6 +69,13 @@ Timescale Cloud or on a self-hosted Postgres server.
   vectorizer
   ```
 
+To validate that your API key is set correctly, run the following SQL query:
+```sql
+SELECT *
+FROM ai.openai_list_models();
+```
+If you get a list of available models you are all set.
```

Comment:
`ai.openai_list_models()` won't work as described on Timescale Cloud.

Reply:
why not?

Reply:
well, you have to run `select set_config('ai.openai_api_key', '<your-key-here>', false);` first